modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3288,12 +3395,25 @@ Relationship type expression cannot possibly be satisfied. (`R1&R2` can never be
 m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
 |Title
 a|The query returns no results because a relationship variable is bound more than once.
-|SeverityLevel
-m|WARNING
+|Description
+a|
+- A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
+- A variable-length relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
```

Comment:
```suggestion
- A variable-length relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result. (%s)
```